indexclient/client.py

```diff
@@ -18,6 +18,7 @@
     "acl",
     "authz",
     "urls_metadata",
+    "uploader",
```

Comment:
why was this change necessary?

Reply:
I guess it is not completely necessary, but it is part of the record schema. If you want to leave it out I will just remove.

Reply:
I would say it's safer to leave it out for now